model/src/main/java/io/streamzi/openshift/dataflow/deployment/TargetState.java

```diff
@@ -232,30 +254,28 @@ private ConfigMap buildEnvVarConfigMaps(ProcessorNode node) {
 
     }
 
-    private ConfigMap buildTopicConfigMaps(ProcessorLink link) {
-
-        String topicName = flow.getName() + "-" + link.getSource().getParent().getUuid() + "-" + link.getSource().getName();
-
-        final Map<String, String> data = new HashMap<>();
-        data.put("name", topicName);
-        data.put("partitions", "20");
-        data.put("replicas", "1");
+    private CustomResource buildTopicCrds(ProcessorLink link) {
```

Comment:
Very picky but shouldn't this be `buildTopicCrs` ?

Reply:
make sense !

or even `buildTopicCr` ?

Reply:
+1